scipy/sparse/tests/test_base.py

```diff
@@ -714,6 +714,7 @@ def test_diagonal(self):
         mats.append([[1],[0],[2]])
         mats.append([[0,1],[0,2],[0,3]])
         mats.append([[0,0,1],[0,0,2],[0,3,0]])
+        mats.append([[1,0],[0,0]])
```

Comment:
Right, but that's my personal preference for prioritizing clean code over the convenience of `git blame`.  One can always dig a little deeper with git if `git blame` doesn't immediately give you the information that you need.

Reply:
BTW, can you join us for the meetup? We will definitely try to talk about things like these and your code style PR.

Reply:
Since this was a pretty small fix, I chose to follow the existing style and minimize the diff. I agree that eventually we should autoformat the whole test suite in one sweep.